### DISCUSSION TITLE
Marketplace: show 3rd party app disclaimer

### DISCUSSION THREAD

**Discussion by yalozhkin** [blockscout/frontend#249] [Ideas]
TBD:

- [x] Design concepts
- [ ] Hand off spec

**Comment by yalozhkin** [comment]
<img width="608" alt="image" src="https://user-images.githubusercontent.com/454947/194055852-9b467e8e-7db9-4ddc-ba44-473637fdc4b5.png">

@berdee @ulyanas @tgladilina @mikhin

**Comment by mikhin** [comment]
Need to add flag to app list so moderator could switch what apps need this disclaimer
